### ISSUE TITLE
[1.16.4 vanilla SMP server] "Health threshold before quit reached. Bot will be stopped." while at full health

### ISSUE THREAD

**Issue by CursedBlackCat** [MrKinau/FishingBot#102]
I'm getting "Health threshold before quit reached. Bot will be stopped." upon login, thus causing the bot to immediately exit. I am at full health at login time.

This happens when I set `auto-quit-before-death` `enabled` to `true`, and happens for all values of `min-health-before-quit`. 

I am playing on 1.16.4, on a vanilla SMP server.

Here is the console log:
```
 java -jar FishingBot-2.7.12.jar
[2020-12-03 19:32:19] [INFO   ] Using language: ENGLISH @ /lang/fb_en.properties
[2020-12-03 19:32:19] [INFO   ] Configuration was loaded: C:\Users\<redacted>\Desktop\FishingBot\config.json
[2020-12-03 19:32:19] [INFO   ] Trying to authenticate using access token...
[2020-12-03 19:32:20] [INFO   ] Authentication successful!
[2020-12-03 19:32:20] [INFO   ] Username: CursedBlackCat
[2020-12-03 19:32:20] [INFO   ] Pinging <redacted> with protocol of MC-1.16.4
[2020-12-03 19:32:20] [INFO   ] Received pong: <redacted>
[2020-12-03 19:32:20] [INFO   ] <redacted> version: 1.16.4 (754), players online: 0
[2020-12-03 19:32:20] [INFO   ] Module "FishingModule" enabled!
[2020-12-03 19:32:20] [INFO   ] Module "HandshakeModule" enabled!
[2020-12-03 19:32:20] [INFO   ] Module "LoginModule" enabled!
[2020-12-03 19:32:20] [INFO   ] Module "ChatProxyModule" enabled!
[2020-12-03 19:32:20] [INFO   ] Module "ClientDefaultsModule" enabled!
[2020-12-03 19:32:20] [INFO   ] Login successful!
[2020-12-03 19:32:20] [INFO   ] Name: CursedBlackCat
[2020-12-03 19:32:20] [INFO   ] UUID: <redacted>
[2020-12-03 19:32:20] [INFO   ] Achieved level: 48
[2020-12-03 19:32:20] [WARNING] Health threshold before quit reached. Bot will be stopped.
```

**Comment by MrKinau** [comment]
Thanks for your issue. This issue was already reported on my Discord and will get fixed in the next release (2.7.14). 

Weird stuff: It will only happen on vanilla servers, if no player is in range of the bot, otherwise it is working.

I'll keep this issue open as long as the 2.7.14 is not released.

**Comment by MrKinau** [comment]
fixed in the latest release: https://github.com/MrKinau/FishingBot/releases/tag/2.7.14
